index.js

```diff
@@ -25,10 +40,17 @@ function def(input, opts) {
 
 module.exports = (input, opts) => {
 	opts = opts || {};
-
-	const fn = process.platform === 'win32' ? win : def;
 	const errors = [];
 
+	let fn;
```

Comment:
Are you guys fine with this or do you prefer something like `let fn = () => Promise.reject('Unsupported environment');`?